Share to Instagram Stories does not work

Instagram Stories is opened with a black screen (no image).

It happens when loading a network image:

```dart
var response = await get("https://images.pexels.com/photos/414171/pexels-photo-414171.jpeg");
Directory tempDir = await getTemporaryDirectory();
File file = File('${tempDir.path}/test.jpg');
File createdFile = await file.create(recursive: true);
if (await createdFile.exists())
    file.writeAsBytesSync(response.bodyBytes);
SocialShare.shareInstagramStory(file.path, "#ffffff", "#000000", "https://deep-link-url");
```

And also when selecting an image through [ImagePicker](https://pub.dev/packages/image_picker):

```dart
File file = await ImagePicker.pickImage(source: ImageSource.gallery);
SocialShare.shareInstagramStory(file.path, "#ffffff", "#000000", "https://deep-link-url");
```
![Instagram Stories Problem](https://user-images.githubusercontent.com/34043575/73882890-a8c86180-4841-11ea-8fd0-7ad1f5b893f1.png)

Hey guys,

I don't know if it is related to the newest Flutter version, but I've just checked it out again and now the code won't even compile. I'm getting the following error:

```
/home/guilherme/Desktop/mobile_app/android/app/src/debug/AndroidManifest.xml Error:
	Attribute provider#androidx.core.content.FileProvider@authorities value=(com.positiv.positiv_mobile_app.com.shekarmudaliyar.social_share) from (unknown)
	is also present at [:open_file] AndroidManifest.xml:14:13-64 value=(com.positiv.positiv_mobile_app.fileProvider).
	Suggestion: add 'tools:replace="android:authorities"' to <provider> element at AndroidManifest.xml to override.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute provider#androidx.core.content.FileProvider@authorities value=(com.positiv.positiv_mobile_app.com.shekarmudaliyar.social_share) from (unknown)
  	is also present at [:open_file] AndroidManifest.xml:14:13-64 value=(com.positiv.positiv_mobile_app.fileProvider).
  	Suggestion: add 'tools:replace="android:authorities"' to <provider> element at AndroidManifest.xml to override.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 13s
```

--- a/README.md
+++ b/README.md
@@ -13,6 +13,20 @@ With this plugin you can share on instagram stories and facebook stories and als
 
 ### Android Configuration
 
+#### Paste the following attribute in the `manifest` tag in the `android/app/src/main/AndroidManifest.xml`:
+
+```
+ 		`xmlns:tools="http://schemas.android.com/tools"`
+```
+
+##### For example:
+
+```
+        <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+                xmlns:tools="http://schemas.android.com/tools"
+                package="your package...">
+```
+
 #### Add this piece of code in the `manifest/application` in the `android/app/src/main/AndroidManifest.xml`:
 
 ```
@@ -20,7 +34,8 @@ With this plugin you can share on instagram stories and facebook stories and als
             android:name="androidx.core.content.FileProvider"
             android:authorities="${applicationId}.com.shekarmudaliyar.social_share"
             android:exported="false"
-            android:grantUriPermissions="true">
+            android:grantUriPermissions="true"
+            tools:replace="android:authorities">
             <meta-data
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/filepaths" />

--- a/README.md
+++ b/README.md
@@ -184,6 +184,10 @@ SocialShare.checkInstalledAppsForShare();
 
 ![copy demo](demogifs/copy.gif)
 
+# Known issues
+
+On Android 6, Share to Instagram Stories may not open Instagram app or load the specified image.
+
 # Buy me a :coffee: !!
 
 ((More coffee == more code) == more fast updates)

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: social_share
 description: Wide variety of sharing options you'll need to share directly to certain well-known apps.
-version: 2.0.3
+version: 2.0.4
 authors:
   - <sskshekar10@gmail.com>
 homepage: https://github.com/ShekarMudaliyar/social_share

